src/helloworld/README.md

```diff
@@ -150,3 +150,45 @@ x0             0x1006e0            1050336
 (gdb) info reg c0
 c0             0x905f400046ec06e000000000001006e0 0x1006e0 [rR,0x1006e0-0x1006ec]
 (gdb)
+```
+
+If a capability (either in memory or a register) is untagged, it is
+either displayed as a simple scalar value without any attributes, or
```

Comment:
Something about null-derived capabilities?

Reply:
I've reworded this a bit and included a specific mention of null-derived capabilities.  Hopefully it is clearer now.